### DISCUSSION TITLE
Make "Write a reply" more obvious

### DISCUSSION THREAD

**Discussion by noah-nuebling** [community/community#38320] [Discussions]
On my repo people often accidentally create new top level comments instead of replying to an existing comment, making the conversation hard to follow.

I'm not sure why this happens but I guess people just don't see the "Write a reply" option sometimes. So if the "Write a reply" option was made more visually prominent that should help the problem.

An example of this problem are [this comment](https://github.com/noah-nuebling/mac-mouse-fix/discussions/101#discussioncomment-3932907), [this comment](https://github.com/noah-nuebling/mac-mouse-fix/discussions/101#discussioncomment-3936408) and [this comment](https://github.com/noah-nuebling/mac-mouse-fix/discussions/101#discussioncomment-3937594), all in the same discussion.

**Comment by vkbo** [comment]
I came here to say this. I'd say most people who come to my discussion page will post a new comment or answer instead of replying to my comment or answer. This quickly becomes impossible to follow if more people jump into the same post. (I mostly use discussions to handle user questions for my project.)

Alternatively, or perhaps even additionally, making it possible for a moderator to move a comment or answer into the correct thread would be very helpful. I usually have to ask the commenter to do it themselves, and then I delete the duplicate.
